asciidoc/modules/con_mg_migrating-applications-to-vertx-http-client.adoc

```diff
@@ -203,7 +203,8 @@ response.end(ar -> {
 ----
 +
 
-On the other hand, we can handle the response with some operations if we want to. You can see the difference in the next examples.
+On the other hand, you can also handle the response with operations like onSuccess, compose, bodyHandler, etc.
+The following examples demonstrate handling responses with the operation onSuccess :
 
 The following example shows an example using HTTP client in {VertX} {v3x} releases:
 
```

Comment:
@jcarranzan Sorry missed these:

The following example shows how to use HTTP client in {VertX} {v3x} releases.

The following example shows how to use HTTP client with the `result()` method in {VertX} {v4}.

Can you please change the comment in both Vert.x 3 and 4 examples from:
//Handler some operation with the response
to 
//Code to handle HTTP response